extensions/Worker.Extensions.Http.AspNetCore.Analyzers/src/DiagnosticDescriptors.cs

```diff
@@ -18,5 +18,8 @@ private static DiagnosticDescriptor Create(string id, string title, string messa
         public static DiagnosticDescriptor CorrectRegistrationExpectedInAspNetIntegration { get; }
             = Create(id: "AZFW0014", title: "Missing expected registration of ASP.NET Core Integration services", messageFormat: "The registration for method '{0}' is expected for ASP.NET Core Integration.",
                 category: Usage, severity: DiagnosticSeverity.Error);
+        public static DiagnosticDescriptor MultipleOutputHttpTriggerWithoutHttpResultAttribute { get; }
+            = Create(id: "AZFW0015", title: "Missing HttpResult attribute in multi-output function", messageFormat: "The return type for function '{0}' is missing an HttpResultAttribute on the HTTP response type property.",
```

Comment:
nit: should say "missing **a** `HttpResultAttribute`"

saying "missing **an** `HttpResult` attribute" would make sense but the attribute name is `HttpResultAttribute` so I think we should use that and that is singular so a make more sense?

Please also make this change in the rule file